### ISSUE TITLE
Consensus-layer Call 87 Agenda

### ISSUE THREAD

**Issue by timbeiko** [ethereum/pm#527]
# Consensus-layer Call 87 Agenda
[prev: call 86](https://github.com/ethereum/pm/issues/521)

Meeting Date/Time: Thursday 2022/5/19 at 14:00 UTC
Meeting Duration 1.5 hours 
[live stream](https://youtu.be/-6dZVes6aWc)

1. Merge
    * testing and general updates
    * [ropsten ttd discussion](https://github.com/ethereum/pm/issues/527#issuecomment-1130287515)
3. Other client updates (if any)
4. Research, spec, etc
    * [0x01 initiated exits](https://github.com/ethereum/pm/issues/527#issuecomment-1124702172)
    * [step deprecation](https://github.com/ethereum/pm/issues/527#issuecomment-1126033489)
6. Open Discussion/Closing Remarks

**Comment by arwer13** [comment]
I would propose to have a brief discussion on "Whether 0x01 withdrawal credentials initiated exits should be added to Shanghai along with the withdrawals" (regarding [post](https://ethresear.ch/t/withdrawal-credentials-exits-based-on-a-generalized-message-bus/12516)).

**Comment by djrtwo** [comment]
I would like to bring up the `step` deprecation to decide if it should be included or not
https://github.com/ethereum/consensus-specs/pull/2856

**Comment by timbeiko** [comment]
I would like to discuss whether to update the Ropsten Merge TTD value. On the last ACD, we agreed to set it to `43531756765713534` such that the fork happens around June 8th. It seems the estimator we used to determine this number was wrong and this value is more likely to occur late-June than around June 8th. 

There are a few options for what we can do: 

1. Keep the same TTD and increase hashrate on Ropsten significantly in the week before June 8th to ensure we hit this value during that week. Initial estimates indicate it's probably not cost-prohibitive to do this, but we should be sure and figure out who can provide mining power. 

2. Use a lower TTD value which we expect to happen around June 8th. While this may seem "obvious", we did get it wrong the first time around! We should make sure that we don't accidentally select a TTD which we end up hitting prior to Bellatrix's activation on the Ropsten Beacon Chain. 

3. A mix of 1+2, use a lower TTD value which, given current hashrate, we expect to hit around, say, June 15th, but then add hashrate to the network to hit it earlier.
